docs/source/examples/annotations.rst

```diff
@@ -15,3 +15,38 @@ Annotations
             b2text = b2.get_tex("x-x_1")
             self.add(dot, dot2, line, b1, b2, b1text, b2text)
 
+.. manim:: ExampleArrow
+    :quality: medium
+    :save_last_frame:
+
+    class ExampleArrow(Scene):
+        def construct(self):
+            dot = Dot(ORIGIN)
+            arrow = Arrow(ORIGIN, [2, 2, 0], buff=0)
+            numberplane = NumberPlane()
+            origin_text = TextMobject('(0, 0)').next_to(dot, DOWN)
+            tip_text = TextMobject('(2, 2)').next_to(arrow.get_end(), RIGHT)
+            self.add(numberplane, dot, arrow, origin_text, tip_text)
+
+.. manim:: ExampleArrowTips
+    :quality: medium
+    :save_last_frame:
+
+    from manim.mobject.geometry import ArrowTriangleTip, ArrowSquareTip, ArrowCircleTip
+    class ExampleArrowTips(Scene):
+        def construct(self):
+            a00 = Arrow(start=[-2, 3, 0], end=[2, 3, 0], color=YELLOW)
+            a11 = Arrow(start=[-2, 2, 0], end=[2, 2, 0], tip_shape=ArrowTriangleTip)
+            a12 = Arrow(start=[-2, 1, 0], end=[2, 1, 0],
+                        tip_shape=ArrowTriangleTip,
+                        tip_style={'fill_opacity': 1, 'stroke_width': 3})
+            a21 = Arrow(start=[-2, 0, 0], end=[2, 0, 0], tip_shape=ArrowSquareTip)
+            a22 = Arrow([-2, -1, 0], [2, -1, 0], tip_shape=ArrowSquareTip,
+                        tip_style={'fill_opacity': 1, 'stroke_width': 3})
+            a31 = Arrow([-2, -2, 0], [2, -2, 0], tip_shape=ArrowCircleTip)
+            a32 = Arrow([-2, -3, 0], [2, -3, 0], tip_shape=ArrowCircleTip,
+                        tip_style={'fill_opacity': 1, 'stroke_width': 3})
```

Comment:
There are separate classes for filled vs. unfilled tips, this is how it currently works:

```suggestion
    from manim.mobject.geometry import ArrowTriangleTip, ArrowSquareTip, ArrowSquareFilledTip,\
                                       ArrowCircleTip, ArrowCircleFilledTip
    class ExampleArrowTips(Scene):
        def construct(self):
            a00 = Arrow(start=[-2, 3, 0], end=[2, 3, 0], color=YELLOW)
            a11 = Arrow(start=[-2, 2, 0], end=[2, 2, 0], tip_shape=ArrowTriangleTip)
            a12 = Arrow(start=[-2, 1, 0], end=[2, 1, 0])
            a21 = Arrow(start=[-2, 0, 0], end=[2, 0, 0], tip_shape=ArrowSquareTip)
            a22 = Arrow([-2, -1, 0], [2, -1, 0], tip_shape=ArrowSquareFilledTip)
            a31 = Arrow([-2, -2, 0], [2, -2, 0], tip_shape=ArrowCircleTip)
            a32 = Arrow([-2, -3, 0], [2, -3, 0], tip_shape=ArrowCircleFilledTip)
```

(I think it makes sense to document custom settings via `tip_style` as well, but I'd rather something like that in the reference manual.)

Reply:
cool. nice!
tip_style in the reference makes sence